assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/codenvy/che.properties

```diff
@@ -294,8 +294,10 @@ che.openshift.liveness.probe.delay=300
 che.openshift.liveness.probe.timeout=1
 che.openshift.workspaces.pvc.name=claim-che-workspace
 che.openshift.workspaces.pvc.quantity=10Gi
-che.openshift.workspace.cpu.limit=1
-# Override memory limit used for openshift workspaces. String, e.g. 1300Mi
+# The amount of memory required for a workspace container to run e.g. 512Mi
```

Comment:
@amisevsk Yes I think it makes sense to have cpu limit and request. Even if we don't need that right now we may need it later. But I would rather add it in second PR.